mantle/kola/tests/misc/network.go

```diff
@@ -84,30 +94,33 @@ NextProcess:
 			c.Fatalf("unexpected number of parts on line: %q in output %q", line, output)
 		}
 		proto := parts[0]
-		portdata := strings.Split(parts[3], ":")
-		port := portdata[len(portdata)-1]
-		pidProgramParts := strings.SplitN(parts[len(parts)-1], "/", 2)
-		if len(pidProgramParts) != 2 {
-			c.Errorf("%v did not contain pid and program parts; full output: %q", parts[6], output)
+		portData := strings.Split(parts[4], ":")
+		port := portData[len(portData)-1]
+		processData := parts[len(parts)-1]
+		pidStr := regexp.MustCompile(`pid=\d+`).FindString(processData)
+		processStr := regexp.MustCompile(`".+"`).FindString(processData)
```

Comment:
So the special sauce here is that we discover `pid` because there is a `pid=` identifier and we discover the process name because it is captured inside of quotes. That last one took me a second to realize. It might be good to mention the approach we're taking in a comment in here.